docs/articles/guides/console-args.md

```diff
@@ -43,14 +43,33 @@ You can also filter the benchmarks by categories:
 
 ## Runtimes
 
-The `--runtimes` or just `-r` allows you to run the benchmarks for selected Runtimes. Available options are: Mono, CoreRT, net46, net461, net462, net47, net471, net472, netcoreapp2.0, netcoreapp2.1, netcoreapp2.2, netcoreapp3.0.
+The `--runtimes` or just `-r` allows you to run the benchmarks for selected Runtimes. Available options are: 
+
+* Clr - BDN will either use Roslyn (if you run it as .NET app) or latest installed .NET SDK to build the benchmarks (if you run it as .NET Core app)
+* Core - if you run it as .NET Core app, BDN will use the same target framework moniker, if you run it as .NET app it's going to use netcoreapp2.0
+* Mono - it's going to use the Mono from `$Path`, you can override  it with `--monoPath`
+* CoreRT - it's going to use latest CoreRT. Can be customized with additional options: `--ilcPath`, `--coreRtVersion` 
+* net46, net461, net462, net47, net471, net472 - to build and run benchmarks against specific .NET framework version 
+* netcoreapp2.0, netcoreapp2.1, netcoreapp2.2, netcoreapp3.0 - to build and run benchmarks against specific .NET Core version
 
 Example: run the benchmarks for .NET 4.7.2 and .NET Core 2.1:
 
 ```log
 dotnet run -c Release -- --runtimes net472 netcoreapp2.1
 ```
 
+Example: run the benchmarks for .NET Core 3.0 and latest .NET SDK installed on your PC:
+
+```log
+dotnet run -c Release -f netcoreapp3.0 -- --runtimes clr core
+```
+
+But same command executed with `-f netcoreapp2.0` is going to run the benchmarks for .NET Core 2.0:
+
+```log
+dotnet run -c Release -f netcoreapp3.0 -- --runtimes clr core
```

Comment:
I guess, it should be `dotnet run -c Release -f netcoreapp2.0 -- --runtimes clr core`

Reply:
it should! good catch!